main.js

```diff
@@ -0,0 +1,57 @@
+var boardArr = [];
```

Comment:
This variable could be part of the `board` object as a property and accessed with `this`. The code would be a bit more reusable that way.